### DISCUSSION TITLE
Is there a way to hide the default error? or at least move it to the bottom?

### DISCUSSION THREAD

**Discussion by Senbonzakura1234** [yoavbls/pretty-ts-errors#18] [Q&A] [answered]
If this can be achieved by your code base change, pls consider to do so.

**Comment by Schmell** [comment]
Agreed! Your error is sooo good it makes the default garbage spew look even worse ( which i didn't think was  even possible).
Thanks for the great extension!!

**Comment by yoavbls** [comment, marked as answer]
Thank you!
Unfortunately, as far as we know there isn't a way to control the order or hide the original errors. Let's continue the discussion there: https://github.com/yoavbls/pretty-ts-errors/issues/3
